packages/venia-concept/src/components/ForgotPassword/FormSubmissionSuccessful/formSubmissionSuccessful.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classify from 'src/classify';
+import Button from 'src/components/Button';
+import defaultClasses from './formSubmissionSuccessful.css';
+// TODO: remove after merging dark theme button classes
+import darkButtonClasses from '../darkButton.css';
+
+class ForgotPassword extends Component {
```

Comment:
This component name is the same as `src/components/ForgotPassword/forgotPassword.js`.

Reply:
Yeah, I will change this, thank you